nano/templates/account_management.tmpl

```diff
@@ -167,6 +167,7 @@
 	{{:helper.link('Department Accounts', 'suitcase', {'state' : 'dep_accounts'}, data.auth != 1 ? 'disabled' : data.state == "dep_accounts" ? 'selected' : null)}}
 	{{:helper.link('System Logs', 'clipboard', {'state' : 'logs'}, data.auth != 1 ? 'disabled' : data.state == "logs" ? 'selected' : null)}}
 	{{if data.state == "crew_accounts"}}
+		<span class='floatRight'>{{:helper.link('Set Passenger Rate', 'gear', {'action' : 'passenger_rate'}, data.auth != 1 ? 'disabled' : null)}}</span>
```

Comment:
To me personally, this would make more sense on the "Payroll" screen, rather than crew accounts (as passenger fees are kinda like a reverse payroll?)